projects/adrv9009/common/adrv9009_bd.tcl

```diff
@@ -57,6 +59,10 @@ ad_ip_instance util_upack2 util_adrv9009_tx_upack [list \
   SAMPLE_DATA_WIDTH $TX_SAMPLE_WIDTH \
 ]
 
+set interpolation_rate 8
```

Comment:
This variable is redundant, we should avoid doing this, just put the raw number in the process call.

Reply:
Done.